lib/complate.js

```diff
@@ -14,10 +14,12 @@ let bundle = new VirtualBundle("./", {
 module.exports = function(snippet) {
 	return new Promise((resolve, reject) => {
 		// TODO: FND has something more sophisticated somewhere
-		let [ imports, macro ] = snippet.split("\n\n", 2);
-		let viewCode = `${imports}\n${rendering}\nmodule.exports = () => render(${macro});`;
+		let delimiter = "\n\n";
```

Comment:
Doesn't this resolve the TODO above? 🤔

Reply:
![cutesily naive](https://media.giphy.com/media/M3a51DMeWvYUo/giphy.gif)

https://github.com/complate/complate-fractal/blob/1da4ee794d0c662d4dc68702d7ce7e880356ce00/src/transpiler.js#L20
(that's still terribly brittle though)